shiptendance/views/index.ejs

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title><%= title %></title>
+    <link rel='stylesheet' href='/stylesheets/style.css' />
+  </head>
+  <body>
+    <h1><%= title %></h1>
```

Comment:
To keep your code DRY, you can pull out the top of these views into a partial and reuse it.